main.py

```diff
@@ -126,7 +133,14 @@ def main():
     color = get_badge_color(coverage)
     temp_badge = 'badge.svg'
     download_badge(coverage, color, temp_badge)
-    upload_file(temp_badge, BUCKET_NAME, BADGE_NAME)
+    upload_file(temp_badge, BUCKET_NAME, 'image/svg+xml', BADGE_NAME)
+    if UPLOAD_COVERAGE_FILE:
+        upload_file(
```

Comment:
I think we need to parse this with a regex and upload the coverage result only. Otherwise it may be a security concern, e.g. in our case the bucket is public so I can see the file names from the report.

Reply:
The coverage bucket is not exactly public. It currently has a policy that allows public GET of objects. I can switch that to just allow public get for svg files only.

Reply:
@safts  agree with you and fixed!